package.json

```diff
@@ -17,13 +17,13 @@
   "dependencies": {
     "@hapi/hoek": "9.x.x",
     "@hapi/statehood": "7.x.x",
-    "uuid": "7.x.x"
+    "uuid": "^8.3.0"
```

Comment:
Any reason not to use `8.x.x`?

Reply:
I thought for external dependencies we typically lock it more, I was surprised this was so open. But happy to do 8.x.x if preferred.

Reply:
I don't have a strong preference. I just noticed that it was different, so I figured I'd ask.

Reply:
FWIW we accepted that syntax when it helped with security issues or depended on a feature only included in a minor. I don't think it's that important. But discussing ^ vs ~ for external deps should probably be on the TSC's agenda, it was virtually a ^ here.